src/ApplyOfflineMap.cpp

```diff
@@ -103,12 +105,23 @@ int ApplyOfflineMap(
 	std::string strVariables,
 	std::string strOutputData,
 	std::string strOutputDataList,
-	std::string strNColName, 
+	std::string strNColName,
+	std::string strInputMesh,
+	std::string strOutputMesh,
```

Comment:
It may be worthwhile to not pollute the original API with so many "required" arguments, for an optional CAAS=True argument. i.e., consider grouping all the new parameters in the end, with default arguments when possible to minimize breakage of existing usage for `ApplyOfflineMap`.